include/faabric/endpoint/Endpoint.h

```diff
@@ -1,9 +1,9 @@
 #pragma once
 
+#include <faabric/proto/faabric.pb.h>
```

Comment:
This is the other side-effect of this change which I think makes much more sense, having the Faabric protobuf-generated headers in the normal include location avoids having to specify a special `include_directories` when using Faabric as a 3rd party dep.

Reply:
Been there :rofl: